lib/node_modules/@stdlib/string/len/test/test.get_emoji_property.js

```diff
@@ -0,0 +1,46 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2020 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var tape = require( 'tape' );
+var constants = require( './../lib/constants.js' );
+var getEmojiProperty = require( './../lib/get_emoji_property.js' );
+
+
+// TESTS //
+
+tape( 'main export is a function', function test( t ) {
+	t.ok( true, __filename );
+	t.strictEqual( typeof getEmojiProperty, 'function', 'main export is a function' );
+	t.end();
+});
+
+tape( 'the getEmojiProperty function returns emoji property', function test( t ) {
+	var out;
+
+	out = getEmojiProperty( 0x23EC );
+	t.strictEqual( out, constants.ExtendedPictographic, 'returns Extended_Pictographic' );
```

Comment:
```suggestion
	t.strictEqual( out, constants.ExtendedPictographic, 'returns expected value' );
```

While some older code has bespoke assertion messages, we've moved to a standardized "returns expected value" message in order to mitigate divergences between the message and what is actually tested. So, e.g., if we changed this test, but forget to update the message, we now have a conflict. By using something fairly generic, we avoid this, and we make it easier to copy-and-paste assertions.